poedit.appdata.xml

```diff
@@ -1,31 +1,76 @@
 <?xml version="1.0" encoding="utf-8"?>
-<components version="0.11">
-  <component type="desktop-application">
+<component type="desktop">
     <id>poedit.desktop</id>
     <name>Poedit</name>
     <summary>Translations editor for gettext</summary>
     <project_license>MIT</project_license>
+    <metadata_license>CC0</metadata_license>
```

Comment:
The spec lists `CC0-1.0` but not `CC0` among valid licenses. While I realize this too is SPDX code, it might make sense to stick to the more specific versioned string.

Reply:
Will fix that!